Zype/Views/TransparentFocusableView.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TransparentFocusableView.swift
+//  DeveloperChallengeAppleTVApp
+//
+//  Created by Advantiss on 10/16/19.
+//  Copyright © 2019 Eugene Lizhnyk. All rights reserved.
```

Comment:
@advantiss425 Is this part necessary? Can we remove it?

Reply:
It needs for this feature.

Reply:
oh... yes, you can remove the lines